### PR TITLE
Fixes previewing

### LINKED ISSUE #5226: ContentPreview became slow
Working on another PR i noticed that a `ContentPreview` (tried with an article type) became slow.

The execution time of the `PreviewController.Render()` seems okay but it takes a long time to be triggered, i just opened the dev tools and saw that a `jquery xhr.send` take a long time, around 1s.

### PR DESCRIPTION
**Pull request by jtkech**
Fixes #5226

### CODE REVIEW

**Comment by Skrypt** [comment]
LGTM

**Comment by Skrypt** [comment]
Should we not have a PreviewAspect or this is too much code for doing the same thing?
At least it would make this more formal instead of guessing that the ContentItem is having a id = -1.

**Comment by jtkech** [comment]
Yes maybe, quickly done, will see tomorrow

File: src/OrchardCore.Modules/OrchardCore.Lucene/Handler/LuceneIndexingContentHandler.cs [hunk 1]
```diff
@@ -26,6 +26,12 @@ public class LuceneIndexingContentHandler : ContentHandlerBase
 
         private Task AddContextAsync(ContentContextBase context)
         {
+            // Check e.g if previewing.
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Lucene/Handler/LuceneIndexingContentHandler.cs]: Could we check the nullability

Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.Lucene/Handler/LuceneIndexingContentHandler.cs]: I'll see, normally we don't need it, a context always has a non null ContentItem, and afterwards in the deferred task it is used as is without null check.

**Pull request merged**